server/controllers/handleQuestItemClicked.js

```diff
@@ -67,32 +53,47 @@ export const handleQuestItemClicked = async (req, res) => {
       return res.json({ addedClick: false, numberAllowedToCollect, success: true });
     } else {
       // Move the quest item to a new random location
+      const promises = [];
```

Comment:
this comment is actually about line 52. I think we should remove that console log, otherwise it would spam cloudwatch

Reply:
Why would it spam cloudwatch? It's only triggered if a very specific condition is met.